src/Symfony/Component/RateLimiter/FixedWindowLimiter.php

```diff
@@ -26,8 +26,16 @@ final class FixedWindowLimiter implements LimiterInterface
 {
     private $id;
     private $limit;
-    private $interval;
     private $storage;
+
+    /**
+     * @var int seconds
```

Comment:
When the description is at the property level, IDE (PHPStorm at least) does not use it to document the related constructor arg when the the constructor signature is displayed for help.  Considering this, would it make sense to rather add it as an `@param` annotation to the constructor? I think it'd be more consistent with what already have in the codebase.

Reply:
This makes sense. 

The constructor takes a `\DateInterval`. That is why I added `int` on this property to be more clear what it contains.

Reply:
👍  Missed it, thanks